CHANGELOG.md

```diff
@@ -4,6 +4,31 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 5.5.0
+### Added
+* Agent: Include `LD_LIBRARY_PATH` if and only if set by command. [#1933](https://github.com/microsoft/onefuzz/pull/1933)
+* Service: Added new functionality to the service port from Python to C#. [#1924](https://github.com/microsoft/onefuzz/pull/1924), [#1938](https://github.com/microsoft/onefuzz/pull/1938), [#1946](https://github.com/microsoft/onefuzz/pull/1946)
+
+### Changed
+* Documentation: Update coverage filtering docs. [#1950](https://github.com/microsoft/onefuzz/pull/1950/files)
+* Agent: Allow the agent to skip reporting directories. [#1931](https://github.com/microsoft/onefuzz/pull/1931)
+* Agent/Supervisor/Proxy: Updated multiple third-party Rust dependencies. [#1935](https://github.com/microsoft/onefuzz/pull/1935)
+* CLI: Deprecate `libfuzzer_coverage` task. [#1944](https://github.com/microsoft/onefuzz/pull/1944/files)
+* CLI: Use consistent logger names. [#1945](https://github.com/microsoft/onefuzz/pull/1945)
+* Service: Updated functionality to the service port from Python to C#. [#1922](https://github.com/microsoft/onefuzz/pull/1922), [$1925](https://github.com/microsoft/onefuzz/pull/1925), [#1947](https://github.com/microsoft/onefuzz/pull/1947)
+
+### Fixed 
+* Agent: Allow old reports to be parsed. [#1943](https://github.com/microsoft/onefuzz/pull/1943)
+* Agent: Remove transitive OpenSSL dependency. [#1952](https://github.com/microsoft/onefuzz/pull/1952)
+* Service: Allow old reports to be parsed. [#1940](https://github.com/microsoft/onefuzz/pull/1940)
+
+## 5.4.1
+This a hotpatch to the 5.4.0 release fixing the parsing failures from old crash reports.
+
+### Fixed
+* Agent: Allow old reports to be parsed [#1943](https://github.com/microsoft/onefuzz/pull/1943)
+* Service: Allow old reports to be parsed [#1940](https://github.com/microsoft/onefuzz/pull/1940)
```

Comment:
```suggestion
* Agent: Include `LD_LIBRARY_PATH` in shared library dependency check if and only if set by command. [#1933](https://github.com/microsoft/onefuzz/pull/1933)
* Service: Allow old reports to be parsed [#1940](https://github.com/microsoft/onefuzz/pull/1940)
```
Should've suggested this wording before (original doesn't make sense without the rest of the PR text).